python/paddle/incubate/autograd/utils.py

```diff
@@ -218,6 +227,37 @@ def get_output_var_list(op):
         ]
 
 
+def get_output_vars_from_comosite(op):
+    """origin op outputs must be mapped into outputs of composite rule."""
```

Comment:
more comments for it, its' some like a trick

Reply:
done